### PR TITLE
Portfolio: add contact form

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
# In this milestone we worked on the following 💪🏼  : 

- [x]  Cloned the repository given and copied the code to a new directory.

 > Additional description of the project and its features.

 - Made sure that :
   -  Name field accepts up to 30 chars.
   - Email field accepts only emails.
   - Message field accepts up to 500 chars.
   - All fields have placeholder text.
   - All fields have the correct type.
   - Don't need to implement any functionality that requires JavaScript.
   - Sticks to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma.

- [x]  Added Linters for the project and fixed errors.

##  Build with : 

  - HTML 5
  - CSS  3
  - Stylelint, Github Actions

------

_THANKS FOR YOUR FEEDBACK 💟_

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+      </div>
+      <div class="contact-container">
+        <h1 class="contact-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h1>
+        <form
+          method="post"
+          class="contact-form"
+        >
```
Comment (medaminedev66) [inline index.html:301]: In this milestone, it is required to add the action attribute to the form. Would you Kindly add it? The action attribute is used to specify where we want to send the form data when the form is submitted. 😃

File: index.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+            <li>
+              <label for="name"></label>
+              <input
+                type="text"
+                id="name"
+                name="user_name"
+                placeholder="Full name"
+                class="input"
+                required
+              />
```
Comment (medaminedev66) [inline index.html]: According to the project requirements, the name input should have `maxlength` attribute. Kindly add it 😃

File: index.html [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+            </li>
+            <li>
+              <label for="email"></label>
+              <input
+                type="email"
+                id="email"
+                class="input"
+                name="user_email"
+                placeholder="Email adress"
+              />
```
Comment (medaminedev66) [inline index.html]: Following the project requirements, All input fields should have the `required` attribute to activate the validation on submit. Would you please fix it? By adding `required`, the user will not be allowed to submit the form without entering email into the input field.

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.contact-btn {
+  color: #fff;
+  background: #ff6b00;
+}
```
Comment (medaminedev66) [inline css/main.css]: For each project, the button (e.g., "Get in touch") is a button tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="contact-container">
+        <h1 class="contact-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
```
Comment (medaminedev66) [inline index.html:296]: In order to stick to the design as much as possible, you are advised to use the same fonts for this heading and all other sections as it appears in the Figma design

Original:
image


Yours:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Hamzaoutdoors 👋, I am **Amine** your reviewer for this PR 😃 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Although you have a nice organized code, it would be better to remove all the comments from your codes to make your project professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Hamzaoutdoors) [reply index.html]: Done ✔️ , Thanks for your feedback @medaminedev66

File: index.html (hunk 2 above)
Comment (Hamzaoutdoors) [reply index.html]: Done ✔️, Thanks for your feedback @medaminedev66

File: index.html (hunk 5 above)
Comment (Hamzaoutdoors) [reply index.html:296]: Done ✔️ , Thanks for your feedback @medaminedev66

File: index.html [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+        </h1>
+        <form
```
Comment (Deepakdanger) [inline index.html:298]: When I click enter after filling the form my data is not saved in form spree. It is important to save users' data in a formspree for future purposes.
Requirements:-
"Collect data by using the [Formspree service.](https://formspree.io/html)"

File: css/main.css [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+ul li:last-child {
+  width: 200px;
+}
+
+.contact-btn {
```
Comment (Deepakdanger) [inline css/main.css:378]: I noticed that while changing your button size, You forget to position your button as per the original website.
[Original]
Capture1
[Yours]
Capture2

File: css/main.css [hunk 8]
```diff
@@ -0,0 +1,383 @@
...
+  color: #fff;
+  background: #ff6b00;
+  width: 20%;
+  cursor: pointer;
```
Comment (Deepakdanger) [inline css/main.css:382]: I noticed that you forget to change something mentioned by the previous reviewer.
For each project, the button (e.g., "Get in touch") is a button tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.
Capture3

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Hamzaoutdoors, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- It would be nice if you have a live version where the project can be viewed by everybody

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**